src/hooks/use-hyperion-framework/getContentResourcesByCriteria.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AnnotationPage,
+  CanvasNormalized,
+  InternationalString,
+} from "@hyperion-framework/types";
+
+export interface LabeledResource {
```

Comment:
Maybe we make a breadcrumb comment here for later to build on top of Hyperion's "ContentResource" type, or whatever it was we were discussing earlier, and figure out concatenating existing types with our own?